apollo-ios/Sources/ApolloAPI/SelectionSet.swift

```diff
@@ -117,6 +117,10 @@ extension SelectionSet {
   @inlinable public static func ==(lhs: Self, rhs: Self) -> Bool {
     return lhs.__data == rhs.__data
   }
+  
+  public var debugDescription: String {
+    return "\(self.__data._data as AnyObject)"
```

Comment:
Should this print more than the data inside the `DataDict`, like the `fulfilledFragments` etc? And are we good with a readable format like provided by casting to `AnyObject` like the screenshot or do we want to manually create an output from keys/values?

<img width="355" alt="Screenshot 2024-07-02 at 12 37 25 AM" src="https://github.com/apollographql/apollo-ios-dev/assets/4425109/1ce7a5f4-68ac-4f00-ae43-991ff5c2a5de">